### ISSUE TITLE
No way to create new custom categories

### ISSUE THREAD

**Issue by socalminstrel** [Cidan/BetterBags#685]
### Addon Version

v0.3.14

### World of Warcraft Version

Retail

### Describe the bug

With one of the most recent versions, there appears no way to create new custom categories. The ability was removed from the main options window and the "Configure Categories" pop-out screen has no clear way to do it. There's no button for it and I've tried right-clicking various elements of the pop-out to no avail.

### Steps to reproduce

Open bag
Check options menu, check Configure Categories pop-out screen

**Comment by Helfull** [comment]
You can use the searchbar to create a custom category based on your search criteria

If you search and press "[Enter]" it will open a pop up that does allow you to create a category for your search.

![image](https://github.com/user-attachments/assets/87b7511d-fb44-4f09-9c9b-872455fae018)

**Comment by AMaribo** [comment]
If you want to create a manual category instead of a search category, drag an item to the bag icon in the top-left corner of the bag.
![i5Z9hdu](https://github.com/user-attachments/assets/46a3618f-6cd2-45d0-b134-c9e2e296a126)

**Comment by socalminstrel** [comment]
Fair enough. These weren't clearly indicated ways to do it, IMO, but obviously there _are_ ways to create custom categories. So maybe this would be better changed to a "Feature Request" to have a button in the Configure Categories pop-out or an indicated region in the pop-out to drag items to create a new custom category.

If there's a way for me to change this from bug to feature request, let me know--I don't see a way in Edit, but I'm not a github expert. Or if a developer wants to change it. Alternatively, I can delete and re-create it as a feature request.

**Comment by AMaribo** [comment]
The category creation UI difficulty/lack of documentation is known and is being worked on.
The old Options way of doing it got removed because it broke and had bugs, but is in the process of being replaced.

**Comment by socalminstrel** [comment]
Gotcha. I appreciate all the responses--I do really like the addon, so it's nice to know how to make new custom categories for the time being.

**Comment by Fizgig1973** [comment]
Is there a way to access custom categories once you have created them? I cannot find any in the menu if I want or need to change things.

**Comment by AMaribo** [comment]
> Is there a way to access custom categories once you have created them? I cannot find any in the menu if I want or need to change things.

Left-click the bag icon, then select `Configure Categories`. Clicking on the little arrow beside any of the categories in that list will bring up the list of items (or search category information) and allow you to add or delete items from the category.
